Styleguide/Areas/MyFeature/Pages/Index.cshtml

```diff
@@ -87,7 +86,7 @@
         </div>
     </div>
 </div>
-<script type="text/javascript" src="~/Static/Styleguide/js/mandelbrot.js" ></script>
+<script type="text/javascript" src="_content/Styleguide/Styleguide/js/mandelbrot.js" ></script>
```

Comment:
hmm, that's odd - is `_content` some sort of RazorClassLibrary pattern?

Reply:
yes, I found this in MS docs